test/testsuite-prepare

```diff
@@ -18,14 +22,16 @@ eval set -- "$args"
 while [ $# -gt 0 ]; do
     case $1 in
     -c|--clean)
-        mock_rpms_opts="$mock_rpms_opts --clean"
+        make_rpms_opts="$make_rpms_opts --clean"
         cockpit_create_opts="--force --no-save"
         install_selinux=no
+        use_make_rpms_rev=no
         ;;
     -q|--quick)
-        mock_rpm_opts="$mock_rpm_opts --quick"
+        make_rpms_opts="$make_rpms_opts --quick"
         ;;
     -v|--verbose)
+        make_rpms_opts="$make_rpms_opts --verbose"
```

Comment:
This line will cause testsuite-prepare to fail. Sadly we cannot pass --verbose to make-rpms, because we parse its output (below).